Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM quay.io/ukhomeofficedigital/alpine:v3.13
```

Comment:
Although this could be throw-away code, we should be using the new standard for this: https://github.com/UKHomeOffice/hocs-search/blob/main/Dockerfile is a good example of this.

Reply:
I've updated to the new standard